xla/service/gpu/gpu_offloading_test.cc

```diff
@@ -168,6 +213,18 @@ TEST_F(GpuCompilerTest, CompiledProgramsCount) {
                           RunHloRematerialization(
                               /*memory_limit_bytes=*/10 * 1024, module.get()));
   ASSERT_TRUE(changed);
+  StreamAttributeAnnotator attr_annotator;
+  TF_ASSERT_OK_AND_ASSIGN(bool changed_attr, attr_annotator.Run(module.get()));
+  EXPECT_TRUE(changed_attr);
+  // Verify that the stream attribute for a copy-start is annotated
+  for (std::string i : {"", ".1", ".2", ".3"}) {
+    const HloInstruction* cp_start =
+        FindInstruction(module.get(), "copy-start" + i);
+    EXPECT_TRUE(cp_start->has_backend_config());
+    TF_ASSERT_OK_AND_ASSIGN(GpuBackendConfig gpu_config,
+                            cp_start->backend_config<GpuBackendConfig>());
+    EXPECT_TRUE(gpu_config.operation_queue_id() > 0);
```

Comment:
Actually this is the line the linter was complaining about. It suggests to use ```EXPECT_GT(gpu_config.operation_queue_id(), 0)```

Reply:
Yes, this is it! Thanks for finding it.

Reply:
I introduced a typo and fixed in the latest commit.